dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/AdvancedSdkCore.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.v2.core
+
+import com.datadog.android.v2.api.SdkCore
+import com.datadog.android.v2.api.context.NetworkInfo
+
+/**
+ * FOR INTERNAL USAGE ONLY. THIS INTERFACE CONTENT MAY CHANGE WITHOUT NOTICE.
+ */
+interface AdvancedSdkCore : SdkCore {
```

Comment:
Maybe instead of Advanced we can rename it InternalSdkCore to discourage people from using it?

Reply:
done. I will also introduce something similar (I mean public, but internal interface) later for writing last view event in RUM. Later we maybe can review such internal APIs usage and find a better approach.